README.md

```diff
@@ -1,9 +1,133 @@
+
+```js
+// create a Vega-Lite example using vg.embed
+vg.embed("#vis-scatter", embedSpec, function(error, result) {
+    // result.view is the Vega View
+});
+```
+
+or
+
+```js
+// create a Vega example using vg.parse.spec
+vg.parse.spec(vgSpec, function(error, chart) {
+    // view is the Vega View
+    var view = chart({el:"#vis-scatter"}).update();
+});
+```
+
+**Step 5** In step 4 callback function, create your tooltip with [`Vega View`](https://github.com/vega/vega/wiki/Runtime#view-component-api). If you have a Vega-Lite visualization, use `vlTooltip()` to create your tooltip. If you have a Vega visualization, use `vgTooltip()` to create your tooltip. For example:
+
+```js
+// create a Vega-Lite example using vg.embed
+vg.embed("#vis-scatter", embedSpec, function(error, result) {
+    // result.view is the Vega View, vlSpec is the original Vega-Lite spec
+    vlTooltip(result.view, vlSpec);
+});
+```
+or
+```js
+// create a Vega example using vg.parse.spec
+vg.parse.spec(vgSpec, function(error, chart) {
+    // view is the Vega View
+    var view = chart({el:"#vis-scatter"}).update();
+    vgTooltip(view);
+});
+```
+Congratulations! Now you should be able to see a tooltip working with your visualization.
+
+## Customizing Your Tooltip
+You can customize the look and the content of your tooltip by passing in an optional `options` parameter to `vlTooltip()` or `vgTooltip()`. For example:
+
+```js
+// in vg.embed callback
+vlTooltip(result.view, vlSpec, options);
+```
+or
+```js
+// in vg.parse.spec callback
+vgTooltip(view, options);
+```
+
+The complete structure of `options` looks like this:
+
+```js
+var options = {
+  showFields: [{ // specify data fields to be shown in the tooltip
+    field: ..., // field name in the dataset
+    fieldTitle: ..., // (optional) field title the tooltip should display
+    type: 'date' | 'number' | 'string',
+    format: timeUnit | string specifier
+  }],
+  offset: { // x and y offset (in pixels) of the tooltip
+    x: 10,
+    y: 10
+  },
+  colorTheme: 'light' | 'dark' // specify a color theme the tooltip should use
+}
+```
+
+You don't have to provide a fully-structured `options` object to customize your tooltip. In most cases, a partially filled-out `options` object should suffice.
+
+### Customize Fields
+By default, tooltip shows all top-level data fields bound to an element. You can use the `options.showFields` array to overwrite this behavior. If the `showFields` array is not empty, only fields in this array will be displayed in the tooltip.
+
+[//]: # (TODO(zening): tooltip fields order: can we recommend which fields should go first and which ones should go later? I feel that the fields that are most frequently changing as the mouse moves, as well as the field that's encoded with color should go first.)
+
+### Customize Tooltip Offset
+By default, tooltip is to the bottom right of the cursor (10px right of and 10px below the cursor). You can use `options.offset` to overwrite this.
```

Comment:
What if I don't want `bottom-right`?  

(If we do not support this yet, I'd say let's create an issue and fix it later since it's not that important)